convert2rhel/repo.py

```diff
@@ -68,3 +74,31 @@ def restore_yum_repos():
 
     if not repo_has_restored:
         loggerinst.info("No .repo files to rollback")
+
+
+def is_eus_repos_available():
+    """"""
+    hardcoded_reposdir = _get_hardcoded_repofiles_dir()
+    reposdir = None
+    if os.path.exists(hardcoded_reposdir) and system_info.has_internet_access:
+        reposdir = hardcoded_reposdir
+
+    return reposdir
```

Comment:
Method implies boolean return but returns a `reposdir` or `None`. I'd rename to `get_eus_repos_available`